src/test/java/core/basesyntax/service/impl/ReadFromFileImplTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.exception.FruitShopException;
+import core.basesyntax.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReadFromFileImplTest {
+    private static final FileReaderService readFromFile = new ReadFromFileImpl();
+    private static final String path = "src/test/resources/Hello.csv";
+    private static final String expectedString = "Hello, mate!";
+
+    @BeforeClass
+    public static void beforeClass() {
+        try {
+            Files.writeString(Paths.get(path), expectedString);
+        } catch (IOException e) {
+            throw new RuntimeException("Check the file path " + path, e);
+        }

```

Comment:
you need this logic only for one test method, so you can remove @BeforeClass and make this method private